src/child-applications/child-apps.js

```diff
@@ -15,21 +15,32 @@ export function getAppStatus(appName) {
 	return app ? app.status : null;
 }
 
-export function declareChildApplication(appLocation, activeWhen) {
-	if (!Loader)
-		throw new Error(`You cannot declare a single-spa child application without first specifying the single-spa Loader. Please call singleSpa.setLoader API before declaring child applications`);
-    if (typeof appLocation !== 'string' || appLocation.length === 0)
-        throw new Error(`The first argument must be a non-empty string 'appLocation'`);
-    if (typeof activeWhen !== 'function')
-        throw new Error(`The second argument must be a function 'activeWhen'`);
-    if (childApps[appLocation])
-        throw new Error(`There is already an app declared at location ${appLocation}`);
-
-    childApps.push({
-        appLocation: appLocation,
-        activeWhen: activeWhen,
+export function declareChildApplication(appLocation, arg1, arg2) {
```

Comment:
This is where the actual changes to the source code begin. Note that spaces -> tabs caused this diff to appear bigger than it actually is.